ME/Controller/BuyFailedViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  buyFailedViewController.swift
+//  ME
+//
+//  Created by 새미 on 2023/09/10.
+//
+
+import UIKit
+import SnapKit
```

Comment:
```suggestion
import UIKit

import SnapKit
```
이것도 코드 컨벤션에 관한 이야기인데,
내장 프레임워크와 서드파티 프레임워크 사이에 공백을 주는 건 어떨까?

결국 컨벤션은 본인의 스타일에 따라 다르겠지만 [StyleShare/swift-style-guide](https://github.com/StyleShare/swift-style-guide#%EC%9E%84%ED%8F%AC%ED%8A%B8) 이것도 한 번 읽어보면 좋을 것 같아.